docs/source/user.rst

```diff
@@ -0,0 +1,254 @@
+User's guide
+============
+
+The purpose of this document is to walk the mead maker through all the
+major operations required to use the software.  Throughout this
+document, this role is referred to as the producer.  This is in
+contrast to the consumer, who plays a significant role in the
+distribution section.
+
+Setup
+-----
+
+``pyment`` is like any other Django app in that it must be deployed in
+order to be useful.  Deployment specifics are beyond the scope of this
+document, but references such as `How to deploy with WSGI
+<https://docs.djangoproject.com/en/1.8/howto/deployment/wsgi/>`_ and
+tutorials such as `How To Deploy a Local Django App to a VPS
+<https://www.digitalocean.com/community/tutorials/how-to-deploy-a-local-django-app-to-a-vps>`_
+are available on the internet.  At the very minimum, the producer will
+need a Django user account in order to access the admin site to
+perform many of the operations listed below.  For more information
+about the admin site, including how to use it, please see `Chapter 6:
+The Django Admin Site
+<http://www.djangobook.com/en/2.0/chapter06.html>`_.
+
+Production
+----------
+
+.. warning:: It is essential to remember that there are many ways to
+	     brew mead, and that this software was written to suit the
+	     particular brewing :doc:`process <brewprocess>` used by
```

Comment:
I know now that "process" and "pull requests" are italicized and a different color because they are links. Unfortunately the color contrast on the yellow background (which works okay elsewhere in the doc because of the white background) isn't high enough to immediately see it's blue instead of black, so it's hard to see it's a link. (The document as a whole doesn't do the "underline when hovered over" thing I am used to seeing to indicate a link. I know that might be a style thing.)